test/CRM/ElifeToken/Token/ArticlesLast7DaysTest.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+require_once __DIR__.'/../../../../CRM/ElifeToken/Token/ArticlesLast7Days.php';
+
+class CRM_ElifeToken_Token_ArticlesLast7DaysTest extends PHPUnit_Framework_TestCase {
+  public static function possibleTypes() {
+    return [
+        ['magazine'],
```

Comment:
values to put here? AFAIK there should be 2:
- VOR and other magazine content
- POA
but I don't know what is the string that will come in as an input for these